nntrainer/layers/layer_devel.h

```diff
@@ -255,7 +255,7 @@ class Layer {
    * @param     Tensor Type : NCHW or NHWC
    */
   void setTensorType(const std::string &values) {
-    if (values.compare("NCHW") || values.compare("nchw")) {
+    if (values.compare("NCHW") == 0 || values.compare("nchw") == 0) {
```

Comment:
Other changes use the ternary operator.
For consistency, why not use a ternary operator rather than an if-else here, too?

Reply:
Yes. Updated to use ternary operator here as well. Thank you.